### ISSUE TITLE
Feature Request: Add algolia to website

### ISSUE THREAD

**Issue by ev1stensberg** [facebook/draft-js#1441]
Would be nice to do a search on the entire site, as the api is quite complex and hard to navigate around if you don't know what API method in particular you're looking for 👍

**Comment by flarnie** [comment]
I like this idea! Not sure how hard it would be to add - would be happy to approve a PR for this though.

**Comment by flarnie** [comment]
Looks relatively easy - https://community.algolia.com/docsearch/

I'll look into this, and I think I know some folks at FB who could help.

**Comment by ev1stensberg** [comment]
I can help out if you want to @flarnie

**Comment by flarnie** [comment]
@ev1stensberg Go for it :)

**Comment by flarnie** [comment]
@ev1stensberg  - Ooops - almost forgot, I already submitted the form to get in touch with Algolia docs search. Not sure if it makes sense for you to work on this too.

**Comment by ev1stensberg** [comment]
@flarnie No, you're right, and also it's more probable it gets done within a senseable amount of time too! 🌠

**Comment by s-pace** [comment]
Hi @ev1stensberg @flarnie ,

Just a followup, your DocSearch index is ready. Let us know if you need help in order to index it:

You're now a few steps away from having it working on your website:
- Copy the following CSS/JS snippets and add them to your page

```html
<!-- at the end of the HEAD -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css" />
<!-- at the end of the BODY -->
<script type="text/javascript" src="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
apiKey: '72fc33da9f0cc939661c5185a541a7aa',
indexName: 'draft-js',
inputSelector: '### REPLACE ME ####',
debug: false // Set debug to true if you want to inspect the dropdown
});
</script>
```

- Add a search input if you don't have any yet, and update the inputSelector value in the code snippet to a CSS selector that targets your input field.
- Optionally customize the look and feel by following the [DocSearch documentation](https://community.algolia.com/docsearch/documentation/)
- You can also check [your configuration in the github repo](https://github.com/algolia/docsearch-configs/blob/master/configs/draft-js.json). Feel free to open pull requests!

Feel free to get back to us if you have any issues or questions regarding the integration.
We'd also be happy to get your feedback and thoughts about DocSearch - so we can continue to improve it.

Have a nice day :)

**Comment by simsim0709** [comment]
Hi, @flarnie 

Are you still working on this? If not, I can help out 😀

**Comment by s-pace** [comment]
Feel free to ping if you need any ✋
